tb_plugin/torch_tb_profiler/profiler/kernel_parser.py

```diff
@@ -20,9 +20,15 @@ def parse_events(self, events):
         events = pd.DataFrame(events)
         events = events.astype({"type": "category", "category": "category", "name": "string"}, copy=False)
         kernels = events[events["category"] == "Kernel"]
```

Comment:
could we merge this line with event.category == "Kernel" line by only adding kernel events?

Reply:
Thanks. Done.